include/jemalloc/internal/tcache_externs.h

```diff
@@ -35,6 +35,11 @@ extern unsigned	global_do_not_change_tcache_nbins;
  */
 extern size_t	global_do_not_change_tcache_maxclass;
 
+/* Default bin info for each bin. */
+extern cache_bin_info_t opt_tcache_bin_info[TCACHE_NBINS_MAX];
+/* Records whether a bin's info is specified by malloc_conf. */
+extern bool opt_bin_info_is_set[TCACHE_NBINS_MAX];
```

Comment:
Maybe avoid the bin_info term in the names (since we will limit it to cache_bin internal soon):
```
opt_tcache_ncached_max
opt_tcache_ncached_max_set
```
Also worth adding a getter for opt_tcache_ncached_max (e.g. `tcache_get_default_ncached_max(ind)`), to emphasize that these opt_ things should only be set during init, and no setter is provided since they should not change afterwards.

Reply:
Naming it as `bin_info` mainly because its type is `cache_bin_info_t`, naming it as `ncached_max` could be a bit confusing. WDYT?

Reply:
Each opt_ should correspond to its opt option name, in this case the tcache.ncached_max. The storage type could get changed later but the option likely will stay.

Reply:
I see. I will use the name `opt_tcache_ncached_max` then.